chore(content): Add Documentation for enforceHistoryTimeToLive (#1523)
camunda/camunda-docs-manual: content/update/minor/719-to-720/_index.md

```

We are looking for maintainers for the template engine extensions. Feel free to reach out to us [via the forum](https://forum.camunda.io/c/camunda-platform-7-topics/39) if you are interested.

# Enforce History Time To Live

> Many of our users have installations that contain resources with null historyTimeToLive. As a result,

Reviewer: ```suggestion
Many of our users have installations that contain model resources (BPMN, DMN, CMMN) with null historyTimeToLive. As a result,
```

> their historic data grow over time and remain uncleaned due to this configuration. In the 7.20 release,

Reviewer: ```suggestion
their historic data grow over time and remain uncleaned due to this configuration. The history data of your executed processes will clutter the database and eventually negatively impact the engine's performance – even for runtime processes. Since the engine uses a relational database, only starting to clean up the history when a vast amount of data has been pilled up is very costly and time-consuming – in some situations, there is even no other solution than truncating tables and losing data. This is why with this release
```

> we decided to render the historyTimeToLive mandatory for new deployments or redeployments.

Reviewer: ```suggestion
we decided to make the historyTimeToLive mandatory for new deployments or redeployments.
```

> To achieve that, there is a new feature flag introduced called `enforceHistoryTimeToLive` that has a
> default value of true.

Reviewer: ```suggestion
```

At the same time, we acknowledge there might be use cases where our users might favour to keep the legacy behaviour despite our recommendation. If that is the case, you can always turn off the feature by setting

> the feature flag to `false`. For more information, checkout the new parameter description under <a href="{{< ref "/reference/deployment-descriptors/tags/process-engine#configuration-properties" >}}">Configuration Properties</a>.

Reviewer: ```suggestion
the feature flag `enforceHistoryTimeToLive` to `false`. For more information, checkout the new parameter description under <a href="{{< ref "/reference/deployment-descriptors/tags/process-engine#configuration-properties" >}}">Configuration Properties</a>.
```
